### ISSUE TITLE
Version 5.26.4 Broken?

### ISSUE THREAD

**Issue by averydelmiller** [YunoHost-Apps/ghost_ynh#222]
### Describe the bug

Looks like the current release is broken. Apparently has failed Yunohost's auto tests. I attempted to install before noticing it is listed as broken, and it failed to build.

### Context

- Hardware: Digital ocean droplet / 1 GB Ram, 25 GB disk
- YunoHost version: 11.1.15
- I have access to my server: Through the webadmin
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: *no*

### Steps to reproduce

Simply installing.

### Expected behavior

N/A

### Logs

[https://paste.yunohost.org/raw/ibukelunuh](https://paste.yunohost.org/raw/ibukelunuh)

**Comment by badrihippo** [comment]
Thanks for the report! Yes, I've seen it flagged as broken for a week or two (whenever that Ghost version came out, I suppose). I'd assumed that meant it was being handled here but I realised now that yours is the first report :sweat_smile:

I'm too caught up this week to debug this, but here's my preliminary analysis for whoever comes on next. I think the significant lines from the given log are as follows:

```
2023-03-28 01:23:15,316: DEBUG - + sudo -u ghost__2 PATH=/opt/node_n/n/versions/node/14/bin:/opt/node_n/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin HOME=/var/www/ghost__2 yarn build --environment/production
2023-03-28 01:23:15,550: DEBUG - yarn run v1.22.19
2023-03-28 01:23:15,627: DEBUG - $ ember build --environment/production
2023-03-28 01:23:24,647: DEBUG - The option '--environment/production' is not registered with the 'build' command. Run `ember build --help` for a list of supported options.
2023-03-28 01:23:31,370: DEBUG - - Building
2023-03-28 01:23:31,371: DEBUG - Environment: development
2023-03-28 01:23:31,371: DEBUG - - Building
2023-03-28 01:23:31,372: DEBUG - - building...
2023-03-28 01:25:38,902: DEBUG - Killed
2023-03-28 01:25:38,946: DEBUG - error Command failed with exit code 137.
2023-03-28 01:25:38,947: DEBUG - info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
2023-03-28 01:25:39,008: DEBUG - + ynh_exit_properly
```

Basically trying to run `yarn run v1.22.19` triggers an `ember build --environment/production` which causes an error. I'm not familiar with `ember` but the `--environment/production` syntax looks a bit suspicious to me (shouldn't it be `--environment production` or `--environment=production`?).

Anyway, if nobody else comes by then I'll come back and test-install + have a proper look at it soon, hopefully sometime next week. (Not a maintainer, but I use Ghost on YunoHost a lot and have helped debug a few issues in the past!)

**Comment by tituspijean** [comment]
There indeed seems to be a typo in the build command, _but_ the keyword here is `Killed`: the server ran out of memory dyring the build.

**Comment by averydelmiller** [comment]
I tried messing with the syntax in line 149 of the install script, changing `--environment/production` to `--environment=production`, and while it didn't return the same error, the installation still failed, saying the yunohost API stopped responding.

**Comment by tituspijean** [comment]
Then try installing from the command line, it will avoid any timeout from the webadmin. :)

**Comment by tituspijean** [comment]
Hi @averydelmiller, the aforementioned PR should fix the issue. You are welcome to try it out and give some feedback. :)

`sudo yunohost app install https://github.com/YunoHost-Apps/ghost_ynh/tree/fix -F`
